redbot/cogs/streams/streams.py

```diff
@@ -760,6 +769,11 @@ async def _get_mention_str(self, guild: discord.Guild) -> Tuple[str, List[discor
         if await settings.mention_here():
             mentions.append("@here")
         can_manage_roles = guild.me.guild_permissions.manage_roles
+        can_mention_everyone = guild.me.guild_permissions.mention_everyone
+        if can_mention_everyone:
+            for role in guild.roles:
+                mentions.append(role.mention)
+                return
         for role in guild.roles:
             if await self.config.role(role).mention():
                 if can_manage_roles and not role.mentionable:
```

Comment:
This is all that needs to be done here:
```suggestion
                if not can_mention_everyone and can_manage_roles and not role.mentionable:
```